pkgs/development/libraries/gstreamer/default.nix

```
@@ -25,7 +25,8 @@
 
   gst-vaapi = callPackage ./vaapi { };
 
-  icamerasrc = callPackage ./icamerasrc { };
+  icamerasrc-ipu6 = callPackage ./icamerasrc { };
+  icamerasrc-ipu6ep = callPackage ./icamerasrc { ipu6-camera-hal = ipu6ep-camera-hal; };
```

```suggestion
  icamerasrc-ipu6ep = callPackage ./icamerasrc {
    ipu6-camera-hal = ipu6ep-camera-hal;
  };
```

Expand this for better legibility.

---

done